### DISCUSSION TITLE
Customised nitter URL

### DISCUSSION THREAD

**Discussion by easypaskk** [FreshRSS/FreshRSS#5111] [General]
I can't find where to personalise my nitter instance. The links are still on nitter.net
Sorry, but I can't find it.

**Comment by Frenzie** [comment]
Could you provide a little more context?

**Reply by easypaskk** [reply]
I have my own self-hosted instance of nitter.
The links in fressrss are always on nitter.net and I am looking for the links in freshrss to be on my own instance.
I haven't found where I can set this up.
I use fressrss 1.20.2

**Reply by Frenzie** [reply]
Where do these links come from?

**Reply by easypaskk** [reply]
I subscribe to a Nitter source with my own instance as url. It is validated as a valid RSS feed.

**Reply by Frenzie** [reply]
That seems like more of a Nitter instance problem rather than something to ask about here.

I suppose it might be interesting for someone to write a generic extension that can do some URL replacements, see https://github.com/FreshRSS/Extensions/blob/481333f4cff47496d4c0c5e20da24a68fd11859d/xExtension-ImageProxy/extension.php#L117-L126

**Reply by easypaskk** [reply]
Yes indeed, I have been looking for an extension to set a custom Nitter URL. Thank you for your help.

**Reply by Frenzie** [reply]
But why are you asking about it here?

**Reply by easypaskk** [reply]
I thought that this function was present on freshrss and I could not find it.
I probably confused it with another application.
